src/build_test.cc

```diff
@@ -622,10 +622,13 @@ void BuildTest::RebuildTarget(const string& target, const char* manifest,
   builder.command_runner_.release();
 }
 
-size_t FakeCommandRunner::CanRunMore() const {
+size_t FakeCommandRunner::CanRunMore(bool jobserver_enabled) const {
   if (active_edges_.size() < max_active_edges_)
     return SIZE_MAX;
 
+  if (jobserver_enabled)
+    return SIZE_MAX;
```

Comment:
Actually ... this should never happen, right? Maybe add `assert(!jobserver_enabled);`.

Or do we have to make sure that `MAKEFLAGS` isn't set when running the unittest? (I'm not sure right now if the parsing actually happens in the tests)

Reply:
Added an `assert()`.